woocommerce/woocommerce
# Move tabs to header and adjust title position

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Moves the tabs to the header and adjusts the title positioning.

<img width="538" alt="Screen Shot 2023-04-12 at 5 20 51 PM" src="https://user-images.githubusercontent.com/10561050/231614531-7bb67262-bf1f-4a60-a33c-554af173072a.png">
<img width="908" alt="Screen Shot 2023-04-12 at 5 14 48 PM" src="https://user-images.githubusercontent.com/10561050/231614534-33fc986a-9edf-43e2-91ae-9512a9b2cbe8.png">


Closes #37686  .



### How to test the changes in this Pull Request:

1. Got to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
2. Under the Features tab make sure to enable `product-block-editor`
3. Visit `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. Check that the tabs now appear in the header
5. Click on various tabs to make sure they work as expected
6. Check that the title is centered in the header
7. Narrow the viewport and check that the title is left aligned (note that the buttons may need to be grouped on mobile to improve this experience cc @jarekmorawski )
